### PR TITLE
Add python3 wheels to live USB ISO

### PR DESCRIPTION
**Pull request by syncom**
We should now able to build subzero straight out of the dev-flabored live system!

### CODE REVIEW

File: live-usb-creator/live_scripts/startup [hunk 1]
```diff
@@ -3,9 +3,11 @@
 # /etc/rc.d/init.d/livesys calls out to this script (/usr/local/bin/startup), meaning it’s run on startup.
 # Note that it does NOT block a TTY/login/bash/etc., hence the nfast_block_shell utility script.
 
-/usr/local/bin/install_protoc
+/usr/local/bin/install_protobuf
 
 /usr/local/bin/install_nfast_tools
 
+ln -Tsf /bin/python3 /bin/python
```
Comment (jknutsen-cashapp) [inline live-usb-creator/live_scripts/startup:10]: Will python3 be the only installed version? Or, does this overwrite a 2.x default?

File: live-usb-creator/README.md [hunk 2]
```diff
@@ -13,15 +13,21 @@ Set the following files in place in the same directory as the `Vagrantfile`.
 * CentOS-7-x86_64-Everything-1908.iso (10G): `curl -O https://vault.centos.org/7.7.1908/isos/x86_64/CentOS-7-x86_64-Everything-1908.iso`
 * kernel-devel-3.10.0-957.12.2.el7.x86_64.rpm (17MB): `curl -L -O http://archive.kernel.org/centos-vault/centos/7.6.1810/updates/x86_64/Packages/kernel-devel-3.10.0-957.12.2.el7.x86_64.rpm`
 * protoc-3.14.0-linux-x86_64.zip (1.6MB): `curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.14.0/protoc-3.14.0-linux-x86_64.zip`
+* six-1.15.0-py2.py3-none-any.whl (11.0kB): `curl -L -O https://files.pythonhosted.org/packages/ee/ff/48bde5c0f013094d729fe4b0316ba2a24774b3ff1c52d924a8a4cb04078a/six-1.15.0-py2.py3-none-any.whl`
+* protobuf-3.14.0-cp36-cp36m-manylinux1_x86_64.whl (1MB): `curl -O -L https://files.pythonhosted.org/packages/fe/fd/247ef25f5ec5f9acecfbc98ca3c6aaf66716cf52509aca9a93583d410493/protobuf-3.14.0-cp36-cp36m-manylinux1_x86_64.whl`
```
Comment (jknutsen-cashapp) [inline live-usb-creator/README.md:17]: Longer term, how safe do we need to be with the library file versioning? Should we be auditing and committing these to an internal archive and fetching them so we don't have dependencies on pythonhosted, kernel.org, etc?

File: live-usb-creator/README.md [hunk 3]
```diff
@@ -13,15 +13,21 @@ Set the following files in place in the same directory as the `Vagrantfile`.
...
 ```bash
-$ shasum -a 256 CodeSafe-linux64-dev-12.50.2.iso CentOS-7-x86_64-Everything-1908.iso kernel-devel-3.10.0-957.12.2.el7.x86_64.rpm protoc-3.14.0-linux-x86_64.zip
+$ shasum -a 256 CodeSafe-linux64-dev-12.50.2.iso CentOS-7-x86_64-Everything-1908.iso \
+kernel-devel-3.10.0-957.12.2.el7.x86_64.rpm protoc-3.14.0-linux-x86_64.zip \
+six-1.15.0-py2.py3-none-any.whl protobuf-3.14.0-cp36-cp36m-manylinux1_x86_64.whl
 23ca2c5fc2476887926409bc69f19b772c99191b1e0cce1a3bace8d1e4488528  CodeSafe-linux64-dev-12.50.2.iso
 bd5e6ca18386e8a8e0b5a9e906297b5610095e375e4d02342f07f32022b13acf  CentOS-7-x86_64-Everything-1908.iso
 a27c718efb2acec969b20023ea517d06317b838714cb359e4a80e8995ac289fc  kernel-devel-3.10.0-957.12.2.el7.x86_64.rpm
 a2900100ef9cda17d9c0bbf6a3c3592e809f9842f2d9f0d50e3fba7f3fc864f0  protoc-3.14.0-linux-x86_64.zip
+8b74bedcbbbaca38ff6d7491d76f2b06b3592611af620f8426e82dddb04a5ced  six-1.15.0-py2.py3-none-any.whl
```
Comment (jknutsen-cashapp) [inline live-usb-creator/README.md:29]: Do these packages have signed manifests that we can check?

**Review by jknutsen-cashapp — COMMENTED** [review]
High-level comments about security around these external packages moving forward. I suspect the QoL gains are worth keeping this as-is for now, but I want to learn more about how you think about this.

File: live-usb-creator/live_scripts/startup (hunk 1 above)
Comment (syncom) [reply live-usb-creator/live_scripts/startup:10]: This overrides the python2 default.

File: live-usb-creator/README.md (hunk 3 above)
Comment (syncom) [reply live-usb-creator/README.md:29]: Some of them do, but ultimately, we are using the `https://url`s as the sources of trustworthiness for these artifacts. (Note that for signed packages, we still need to have public keys to trust.)

**Review by oreparaz — APPROVED** [review]

**Pull request merged**